packages/documentation/src/stories/components/link/link.snapshot.stories.ts

```diff
@@ -0,0 +1,37 @@
+import type { Args, StoryContext, StoryObj } from '@storybook/web-components';
+import meta, { Default } from './link.stories';
+import { html } from 'lit';
+import { bombArgs } from '@/utils';
+
+const { id, ...metaWithoutId } = meta;
+
+export default {
+  ...metaWithoutId,
+  title: 'Snapshots',
+};
+
+type Story = StoryObj;
+
+export const Link: Story = {
+  render: (_args: Args, context: StoryContext) => {
+    return html`
+      <div>
+        ${['bg-white', 'bg-dark'].map(
+          bg => html`
+            <div class="${bg} d-flex flex-column gap-regular p-regular mt-regular">
```

Comment:
Here you also have to set a `data-color-mode="light"` or `data-color-mode="dark"` attribute so that the correct set of token is used.